examples/chat/module/rust/src/lib.rs

```diff
@@ -30,6 +30,29 @@ mod proto {
     include!(concat!(env!("OUT_DIR"), "/oak.examples.chat.rs"));
 }
 
+oak::entrypoint_command_handler!(main => Main);
+
+/// Main entrypoint of the chat application.
+///
+/// This node is in charge of creating the other top-level nodes, but does not process any request.
+struct Main;
+
+impl oak::CommandHandler<ConfigMap> for Main {
+    fn handle_command(&mut self, _command: ConfigMap) -> anyhow::Result<()> {
+        let grpc_channel =
+            oak::grpc::server::init("[::]:8080").expect("could not create gRPC server pseudo-Node");
+        oak::node_create(
```

Comment:
Do you have thoughts on how this pattern could be adapted to support the case where the router node needs initial configuration information (e.g. routing decisions are based on info received in the config map)?

Reply:
Yes, that should be doable with the Either type I mentioned elsewhere, more on that in a future PR.

Reply:
I was more thinking about channels rather than types. The current implementation sends the grpc invocation channel to the router as the initial channel, so it does not have a channel over which it could send the init message to the router. It could create another init channel, and include the grpc invocation channel handle in that message, but then we are back to the router node needing to read messages from two different channels.